deploy/helm/minikube-dev.yml

```diff
@@ -87,9 +84,21 @@ magda-core:
       pullPolicy: IfNotPresent
       imagePullSecret: false
 
+  openfaas:
```

Comment:
Here we probably need to config openfaas different for local testing (previously forgot about it):
```yaml
openfaas:
  faasnetes:
    imagePullPolicy: Always
```
This will make openfaas always re-pull the docker image by default